firefox-ios/firefox-ios-tests/Tests/XCUITests/DataManagementTests.swift

```diff
@@ -8,11 +8,13 @@ class DataManagementTests: BaseTestCase {
     func cleanAllData() {
         navigator.goto(WebsiteDataSettings)
         mozWaitForElementToExist(app.tables.otherElements["Website Data"])
+        mozWaitForElementToNotExist(app.activityIndicators.firstMatch)
         // navigator.performAction(Action.AcceptClearAllWebsiteData)
         // We need to fix the method in FxScreenGraph file
         // but there are many linter issues on that file, so this is a quick fix
         app.tables.cells["ClearAllWebsiteData"].staticTexts["Clear All Website Data"].waitAndTap(timeout: TIMEOUT)
         app.alerts.buttons["OK"].waitAndTap(timeout: TIMEOUT)
+        mozWaitForElementToNotExist(app.alerts.buttons["OK"])
```

Comment:
The count for number of websites may not be updated when the next line (XCTAssertEqual) is executed.